Python/optimizer.c

```diff
@@ -394,6 +394,34 @@ executor_traverse(PyObject *o, visitproc visit, void *arg)
     return 0;
 }
 
+static PyObject *
+get_jit_code(PyObject *self, PyObject *Py_UNUSED(ignored))
+{
+#ifndef _Py_JIT
+    PyErr_SetString(PyExc_RuntimeError, "JIT support not enabled.");
+    return NULL;
+#else
+    if (!Py_IS_TYPE(self, &_PyUOpExecutor_Type)) {
+        PyErr_SetString(PyExc_TypeError, "get_jit_code() requires a uop_executor object.");
+        return NULL;
+    }
+    _PyExecutorObject *executor = (_PyExecutorObject *)self;
+    if (executor->jit_code == NULL || executor->jit_size == 0) {
+        PyErr_SetString(PyExc_ValueError, "No JIT code available.");
```

Comment:
This could return an empty string instead of a error

Reply:
I think the exception makes more sense -- though maybe you should only check for `jit_code == NULL`.